symfony/webpack-encore-bundle/1.0/webpack.config.js

```diff
@@ -1,5 +1,9 @@
 var Encore = require('@symfony/webpack-encore');
 
+// Optional: sync the NODE_ENV value with the current webpack environment
+// It's useful when you configure tools like PostCSS in their own config files instead of in webpack.config.js.
```

Comment:
I’d change this to:

It’s useful if you need to read the environment from some other config file

The key difference is that I want people to know that this change doesn’t *cause* any behavior to change, it’s just something you can use if you want yourself.

Reply:
I made the change you proposed. But please, note that if you think this is a rare edge-case and that we shouldn't add this to the config file because it will be confusing to newcomers ... that's perfectly find for me.

You are an expert on Webpack/Encore/JS/etc. and I'm not ... so your decision is the important one. Thanks!